internal/command/postgres/config_update.go

```diff
@@ -251,7 +251,13 @@ func updateStolonConfig(ctx context.Context, app *api.AppCompact, leaderIP strin
 	if !force {
 		// Query PG settings
 		pgclient := flypg.NewFromInstance(leaderIP, dialer)
-		settings, err := pgclient.ViewSettings(ctx, keys)
+
+		_, flex := os.LookupEnv("FORCE_FLEX")
```

Comment:
What does this do?

Reply:
When running in development `app.ImageDetails.Repository == "flyio/postgres-flex"` will not be truthy, so you can bypass that check with the `FORCE_FLEX` env variable.